client/my-sites/upgrades/checkout-thank-you/google-voucher/index.jsx

```diff
@@ -112,7 +112,7 @@ class GoogleVoucherDetails extends Component {
 			<div className="google-voucher__initial-step">
 				<PurchaseButton
 					onClick={ this.onGenerateCode }
-					text={ this.props.translate( 'Generate Code' ) } />
+					text={ this.props.translate( 'Generate code' ) } />
```

Comment:
Warning: 18 translations will be lost with this change.
Alternate string suggestion: 
* `translate( 'Generate Backup Codes' )`&mdash; translations: **30**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-live-course-advertising)